theano/tensor/basic.py

```diff
@@ -4057,6 +4057,9 @@ class Tile(Op):
     <http://docs.scipy.org/doc/numpy/reference/generated/numpy.tile.html>`_
     """
     def __init__(self, ndim):
+        warnings.warn((
+            "Tile op is deprecated, use tile function instead."),
+                      stacklevel=3)
```

Comment:
This was successfully deleted in one of your commits but an ulterior one restored it.